specs/IsSmartScreenRequired

```diff
@@ -0,0 +1,94 @@
+
+It is much easier to indicate if individual WebView2s require SmartScreen than to have an end developer explicitly manage if SmartScreen should be enabled as a whole, especially when its different sets of WebView2s in different processes (like Excel's set of WebView2s and Word's set of WebView2s) all sharing the same user data folder.
+In this document we describe the new setting. We'd appreciate your feedback.
+
+
+# Description
+You can use CoreWebView2Settings.IsSmartScreen to control SmartScreen. SmartScreen is enabled or disabled per browser process, so all WebView2 applications sharing the same user data folder path also share SmartScreen being enabled or disabled.
+If CoreWebView2Setting.IsSmartScreenRequired is true for any CoreWebView2 in its associated CoreWebView2Environment, then SmartScreen is enabled. If CoreWebView2Setting.IsSmartScreenRequired is false for all CoreWebView2s in their CoreWebView2Environment, then SmartScreen is disabled.
+The default value for `IsSmartScreenRequired` is true if SmartScreen is enabled when the CoreWebView2 is created and false if SmartScreen is disabled when the CoreWebView2 is created. The value doesn't change if SmartScreen is enabled or disabled later.
+
+Changes to `IsSmartScreenRequired` take effect on the next navigation or download.
+
+
+# Examples
+```cpp
+wil::com_ptr<ICoreWebView2Settings> m_webViewSettings;
+void SettingsComponent::ToggleSmartScreenRequired()
+{
+    wil::com_ptr<ICoreWebView2Settings11> coreWebView2Settings11;
+    coreWebView2Settings11 =
+                m_webViewSettings.try_query<ICoreWebView2Settings11>();
+    if(coreWebView2Settings11) 
+    {
+        BOOL enabled;
+        CHECK_FAILURE(coreWebView2Settings11->get_IsSmartScreenRequired(&enabled));
+        CHECK_FAILURE(coreWebView2Settings11->put_IsSmartScreenRequired(enabled ? FALSE : TRUE));
+    }
+}
+```
+
+```c#
+void ToggleSmartScreenRequired()
+{
+    var settings = webView2Control.CoreWebView2.Settings;
+    settings.IsSmartScreenRequired = !settings.IsSmartScreenRequired;
+}
+```
+
+# Remarks
+
+# API Notes
+
+See [API Details](#api-details) section below for API reference.
+
+# API Details
+
+## Win32 C++
+```cpp
+[uuid(d667d3a7-c1b7-479f-8833-db7547df6687), object, pointer_default(unique)]
+interface ICoreWebView2Settings11 : ICoreWebView2Settings10 {
+  /// SmartScreen helps you identify reported phishing and malware websites  
```

Comment:
Unclear "you" here. I think it should be "users" and "them".

Reply:
Does "them" mean webview instances, or customers?  There are two instances of "you" in this sentence neither of which refer to developers who will be reading this.  Maybe "SmartScreen helps webviews identify reported phishing and malware websites and also helps users make informed decisions about downloads."

Reply:
Malware websites shows an error page to the end user. Suspicious downloads reports information to the end user (if the app is using the default download UX) and the end developer (via the Downloads API).